### PR TITLE
Float release SEMVER up to major and minor tags

### PR DESCRIPTION
**Pull request by RedbackThomson**
## Description
When tagging with `vX.Y.Z`, the `vX` and `vX.Y` tags will now also be updated to the latest version. This means we are now able to offer customers with major, minor and patch specific tags.

## Testing
Ran this on our development CI/CD pipeline

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: codebuild/scripts/release_tag.sh [hunk 1]
```diff
@@ -18,10 +18,10 @@ function retag_image()
   docker push $image:$new_tag
 }
 
-CODEBUILD_GIT_TAG="$(git describe --tags --exact-match 2>/dev/null)"
+GIT_TAG="$(git describe --tags --exact-match 2>/dev/null)"
```
Comment (cadedaniel) [inline codebuild/scripts/release_tag.sh]: Will we have more than one tag now? e.g. v1.1 -> v1.1.6 -> commit sha?

Comment (RedbackThomson) [reply codebuild/scripts/release_tag.sh]: In git we will only have `vX.Y.Z` tags, but in the containers and artifacts we will have each of the types.

Comment (cadedaniel) [reply codebuild/scripts/release_tag.sh]: Ah, I see (although I think you mean that git will only have `vX.Y` tags?)

Comment (RedbackThomson) [reply codebuild/scripts/release_tag.sh]: Git will only have `rX.Y` branches, but all tags and releases will be `vX.Y.Z`

**Pull request merged**